test/context_list/utils.js

```diff
@@ -0,0 +1,32 @@
+const { assert } = require('chai');
+
+const { parseDataTrack } = require('../../lib/context_list/utils');
+
+describe('parse data-track', () => {
+  it('should return empty object if no data-track attributes provided', () => {
+    const options = {
+      pep: 'coc',
+      nothing: 'related',
+    };
+
+    assert.isEmpty(parseDataTrack(options));
+  });
+
+  it('should return data-track related attributes', () => {
+    const result = {
+      'data-track': 'test',
+      'data-track-category-id': '1',
+      'data-track-cats-and-dogs': 'fishes',
+    };
+
+    const options = {
+      ...result,
+      bird: 'animal',
+      data: 'value',
+      'some-random-data-track': 'value',
+      'data-tracright': 'kek',
+    };
+
+    assert.deepEqual(parseDataTrack(options), result);
```

Comment:
I would inline `options` and `result` - I try to have as little logic in tests as possible

Reply:
Isn't this way more readable?

Reply:
It's not too bad if you span it over multiple lines. I like it better when you instantly see what is happening 

- assert.deepEqual 
- - function call
- - - function args
- - expected

Not sure if the others see it the same way but even if you don't want to inline: Don't merge result into options. Hardcode it.